recipes/mysql.rb

```diff
@@ -16,7 +16,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-node.default['osl-keepalived']['virtual_ipv4'] = %w(140.211.9.47/24)
+node.default['osl-keepalived']['virtual_ipv4'] = %w(140.211.9.47/24
+                                                    10.1.0.86/23)
```

Comment:
This won't work because the backend IP will be on a different interface. You need to have two separate instances for the public and backend IP addresses.